ui/src/app/apps/apps.service.ts

```diff
@@ -102,6 +102,14 @@ export class AppsService {
 
   }
 
+  unregisterMultipleApps(appRegs: AppRegistration[]): Observable<Response[]> {
+    const observables: Observable<Response>[] = [];
+    for (const appReg of appRegs) {
+      observables.push(this.unregisterApp(appReg));
+    }
+    return Observable.forkJoin(observables);
```

Comment:
While using `forkJoin` is not a new thing in this class, doesn't this have a problem that if one `Observable` in forkjoin fails, then whole `Observable` errors and rest in forkjoin are not even tried. Probably not an issue for this PR but we should think about how this behaves when things go south.